Java/tests/src/test/java/com/microsoft/recognizers/text/tests/TestCase.java

```diff
@@ -2,10 +2,12 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
-import java.time.LocalDateTime;
+import java.time.*;
```

Comment:
Use StyleCheck plugin to ensure coding standard.

Reply:
I have applied the StyleCheck suggestions.